src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -7,8 +7,10 @@
 from . import ApiController, RESTController, \
     allow_empty_body
 from .. import mgr
-from ..exceptions import DashboardException
+from ..exceptions import InvalidCredentialsError, UserDoesNotExist
 from ..services.auth import AuthManager, JwtManager
+from ..settings import Settings
+from . import ApiController, ControllerDoc, EndpointDoc, RESTController, allow_empty_body
```

Comment:
```suggestion

```

Reply:
ControllerDoc, EndpointDoc - this needs to be removed as the Document REST API PR is not backported https://github.com/ceph/ceph/pull/36347

Reply:
Thanks @aaSharma14 Will remove that.